mindsdb/integrations/handlers/autosklearn_handler/requirements.txt

```diff
@@ -0,0 +1,3 @@
+auto-sklearn
+mindsdb-evaluator>=0.0.6
```

Comment:
This package seems like it's not being used yet?

Reply:
Are you talking about `mindsdb-evaluator`? The reason I have added this as a dependency, Patricio, is because the previous versions of this package used an old version of `sklearn`, which is not compatible with `auto-sklearn`. You might remember that I opened a PR on `mindsdb-evaluator` to resolve this?

Reply:
Aah yes, I do remember now, that makes sense!